essentials/src/api/subxt_wrapper.rs

```diff
@@ -645,7 +662,11 @@ async fn subxt_get_hrmp_content(
 
 async fn subxt_get_host_configuration(api: &OnlineClient<PolkadotConfig>) -> Result {
 	let addr = subxt::dynamic::storage_root("Configuration", "ActiveConfig");
-	let value = api.storage().at_latest().await?.fetch(&addr).await?.unwrap().to_value()?;
+	let value = match api.storage().at_latest().await?.fetch(&addr).await? {
```

Comment:
nit: Do we want to fetch the `ActiveConfig` from the latest block, or from a dedicated block hash to keep in line with the other functions?

Reply:
In that particular case we're good with the latest.